ansible/roles/test/files/acstests/everflow_tb_test.py

```diff
@@ -138,10 +139,17 @@ def runSendReceiveTest(self, pkt2send, src_port, destination_ports):
         #if (scapy_pkt[scapy.IP].tos >> 2) != self.session_dscp:
         #    return False
 
-        payload = str(scapy_pkt[scapy.GRE].payload)[22:]
+        payload = str(scapy_pkt[scapy.GRE].payload)
+
+        if self.hwsku in ["ACS-MSN2700", "ACS-MSN2100", "ACS-MSN2410"]:
+            payload = str(scapy_pkt[scapy.GRE].payload)[22:]
+
         inner_pkt = scapy.Ether(payload)
 
-        return dataplane.match_exp_pkt(pkt2send, inner_pkt)
+        masked_inner_pkt = Mask(inner_pkt)
+        masked_inner_pkt.set_do_not_care_scapy(scapy.IP, "chksum")
```

Comment:
In case that test packet is a TCP packer, we need to ignore TCP checksum as well.